examples/step-35/doc/intro.dox

```diff
@@ -143,6 +142,33 @@ projection methods that we have just described:
 </ul>
 
 
+<p>
+The standard incremental scheme and the rotational incremental scheme were first considered by van Kan in
+<ul>
+  <li> J. van Kan, A second-order accurate pressure-correction scheme for viscous incompressible flow, SIAM J. Sci. Stat. Comput. 7 (3) (1986) 870–891
+</ul>
+and is analyzed by Guermond in
+<ul>
+  <li> J.-L. Guermond, Un résultat de convergence d’ordre deux en temps pour
+                       l’approximation des équations de Navier–Stokes par une technique de projection incrémentale,
+       ESAIM: Mathematical Modelling and Numerical Analysis, vol. 33, no. 01, pp. 169–189, 1999
```

Comment:
This will be printed without endlines. Want to put the title of the paper in quotes? Same above and below.